client/blocks/login/login-form.jsx

```diff
@@ -428,6 +430,33 @@ export class LoginForm extends Component {
 		);
 	}
 
+	renderAlreadyLoggedInPrompt() {
+		const { isLoggedIn, currentUserName } = this.props;
+		const { isFormDisabledWhileLoading } = this.state;
+
+		return (
+			! isFormDisabledWhileLoading &&
+			isLoggedIn && (
+				<Notice status="is-info" showDismiss={ false } icon="lock">
+					{ this.props.translate(
+						'You are already logged in as {{em}}%(email)s{{/em}}. ' +
+							'{{a}}Click here{{/a}} to continue as that user, or log in as ' +
+							'a different user below.',
+						{
+							args: {
+								email: currentUserName,
+							},
+							components: {
+								a: <a href="/" />,
```

Comment:
The login URL can have a `redirect_to` parameter which specifies a different target URL than `/`. It should be used to continue as current user. The `login` Redux state contains this URL, and it's been already validated agains "open redirect" attacks.

Reply:
Oh, thanks, I'd forgotten about the URL parameter! And thanks for letting me know where to grab it from safely 👍

Reply:
The fields in `state.login` don't appear to be populated when first accessing the login page. As far as I can tell, they're only populated as the result of a `LOGIN_REQUEST`, which doesn't get issued until a login is attempted. The success response to this is what returns the sanitised URL.

I'm not sure what the best way of retrieving the sanitised redirect URL is, in this case. Any thoughts?